tests/functional/flakes/build-paths.sh

```diff
@@ -56,6 +56,18 @@ cat > $flake1Dir/flake.nix <<EOF
     a12 = self.drvCall.outPath;
 
     a13 = "\${self.drvCall.drvPath}\${self.drvCall.outPath}";
+
+    a14 = with import ./config.nix; mkDerivation {
+      name = "dot-installable";
+      outputs = [ "foo" "out" ];
+      meta.outputsToInstall = [ "out" ];
+      buildCommand = ''
+          mkdir \$foo \$out
+          echo "foo" > \$foo/file
+          echo "out" > \$out/file
+      '';
+      outputSpecified = true;
+    };
```

Comment:
Idiomatically, `outputSpecified` doesn't exist on the top attr.
```suggestion
    a14 = with import ./config.nix; let 
      top = mkDerivation {
        name = "dot-installable";
        outputs = [ "foo" "out" ];
        meta.outputsToInstall = [ "out" ];
        buildCommand = ''
            mkdir \$foo \$out
            echo "foo" > \$foo/file
            echo "out" > \$out/file
        '';
      };
    in top // {
      foo = top.foo // {
        outputSpecified = true;
      };
    };
```